MapboxNavigation-SPM.xcodeproj/project.pbxproj

```diff
@@ -1376,8 +1376,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/mapbox/MapboxGeocoder.swift.git";
 			requirement = {
-				kind = upToNextMajorVersion;
-				minimumVersion = 0.14.0;
+				kind = exactVersion;
```

Comment:
Change is needed to prevent build issues (starting from MapboxGeocoder no longer supports iOS 11.x).